src/gui/org/deidentifier/arx/gui/view/impl/menu/DialogOrderSelection.java

```diff
@@ -199,7 +199,7 @@ private boolean isValidDataType(DataType<?> type, String[] values) {
      * @param charset TODO
      * @return
      */
-    private String[] loadArray(String file, Charset charset) {
+    private ArrayList<String> loadFile(String file, Charset charset) {

```

Comment:
Yes, please. Also, I think that the call to retainAll will not be needed anymore, after this patch has been revised.